src/App.jsx

```diff
@@ -1,107 +1,28 @@
 import './App.scss';
-
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId)
+    || null;
+}
+
+function getCommentsByPostId(postId) {
+  return commentsFromServer.filter(comment => comment.postId === postId)
+  || [];
```

Comment:
I think you don't need this conditional because the filter method returns an empty array anyways.
```suggestion
  return commentsFromServer.filter(comment => comment.postId === postId);
```